errors/multi_test.go

```diff
@@ -57,6 +57,31 @@ func TestAddToMultiErr(t *testing.T) {
 	}
 }
 
+func TestMultiErrContains(t *testing.T) {
+	var (
+		myErrNotFound = Wrap(ErrNotFound, "test")
+		myErrState    = Wrap(ErrState, "test")
+		myErrMsg      = Wrap(ErrMsg, "test")
+	)
+	specs := map[string]struct {
+		err error
+		exp *Error
+	}{
+		"A wrapped multierr should pass an is check": {
+			err: Wrap(Append(Append(myErrNotFound, myErrState), myErrMsg), "wrapped"),
```

Comment:
What about the case:
`Append(Wrap(Append(Wrap(`. This is two levels and the default behavior with proposed `Appendf`. Let's make sure it works well. Note the Wrap *between* Appends, which I would suspect *could* cause trouble unless properly coded. Let's make sure it works properly now and in the face of future changes

Reply:
Sure thing